pkg/database/database.go

```diff
@@ -315,6 +316,11 @@ func callbackIncrementMetric(ctx context.Context, m *stats.Int64Measure, table s
 					_ = scope.Err(fmt.Errorf("failed to parse realm_id: %w", err))
 					return
 				}
+
+				if raw >= 1<<bits.UintSize-1 {
```

Comment:
https://golang.org/pkg/math/#pkg-constants

Reply:
Right, that tells me the max 32 and 64-bit uints, but it doesn't tell me what the max value of `uint` is on the system, since `int` and `uint` vary based on the architecture (32 vs 64 bit)